src/undo.h

```diff
@@ -86,21 +86,87 @@ struct ScUndoData
 /** Undo information for a CBlock */
 class CBlockUndo
 {
+    /** Magic number read from the value expressing the size of vtxundo vector.
+     *  It is used for distinguish new version of CBlockUndo instance from old ones.
+     *  The maximum number of tx in a block is roughly MAX_BLOCK_SIZE / MIN_TX_SIZE, which is:
+     *   2M / 61bytes =~ 33K = 0x8012
+     * Therefore the magic number must be a number greater than this limit. */
+    static const uint16_t _marker = 0xfec1;
+
+    /** memory only */
+    bool isNewVersion;
```

Comment:
Here I'd go for something more telling, like "fIncludeSidechainAttributes"

Reply:
ok